src/main/resources/init.sh

```diff
@@ -77,11 +77,11 @@ status)
                 exit 0
             else
                 printf "%s\n" "Warning, Pid $PID appears to not correspond to service $SERVICE"
-                exit 0
             fi
         fi
-        printf "%s\n" "Process dead but pidfile exists"
-        exit 1
+        printf "%s\n" "Process dead but pidfile exists. Cleaning up."
+        rm -f $PIDFILE
+        exit 3
```

Comment:
We don't want `init.sh status` to attempt killing a PID that's not the service in question. `status` is called at the beginning of every other command; it seems like the most sensible spot unless we want to pull out pid file handling to a separate function completely